src/router.ts

```diff
@@ -3,8 +3,10 @@ const routes = {
     "projects.new": () => `/projects/new`,
     "projects.edit": ({ id }: { id: string }) => `/projects/${id}`,
     report: () => "/report",
-    dataEntry: () => "/data-entry",
-    "dataEntry.edit": ({ id }: { id: string }) => `/data-entry/${id}`,
+    actualValues: () => "/actual-values",
+    // "dataEntry.edit": ({ id }: { id: string }) => `/dataEntry/${id}`,
```

Comment:
Actually, we will pass the org unit (project) in the route. @laurapareja  can you please change it to pass the org unit even though we afterwards use a hardcoded org unit and dataset?

Reply:
Same approach apply for both actualvalues and targetvalues